examples/mod-source-record-storage/recordCollection.sample

```diff
@@ -447,6 +447,9 @@
           ]
         }
       },
+      "externalIdsHolder": {
+        "instanceId": "54cc0262-76df-4cac-acca-b10e9bc5c79a"
```

Comment:
Please rename it from "instanceId" into "inventoryInstanceId" to be able to differ instance from mod-inventory/mod-marccat